tests/openassetio-python/CMakeLists.txt

```diff
@@ -195,35 +195,41 @@ openassetio_add_pytest_target(
 #-----------------------------------------------------------------------
 # CTest setup fixtures
 
-# Fixture: create Python environment.
-openassetio_add_test_fixture_target(openassetio-python-venv)
+# Optional fixture dependency to create a Python environment in the
+# install tree.
+set(_venv_if_enabled )
```

Comment:
[consider][nit-pick] trailing space

Reply:
I put that to mean "explicitly blank" (seen that pattern elsewhere), but it works without the space, so happy to remove.

Reply:
Ahh cool, sorry that does make sense, I hadn't groked the implication, but I'm a CMake muppet. Leave as is!